packages/react-error-overlay/src/overlay.js

```diff
@@ -156,6 +160,10 @@ function crash(error: Error, unhandledRejection = false) {
   }
   consumeError(error, unhandledRejection, CONTEXT_SIZE)
     .then(ref => {
+      if (ref == null) {
+        console.warn('react-error-overlay ignored:', error);
```

Comment:
What would be a better message?

```
react-error-overlay could not deduce if this error was from your code. Since we can't display anything useful, we're giving you this warning instead.
This error is most likely a bug with the package you see in the stack trace, however, it may be caused indirectly by something you are calling within the package.
Try searching or opening an issue with the relevant package.
${error}
```